app/lib/swagger_docs.rb

```diff
@@ -300,7 +300,9 @@ def schema
                 },
                 net_employment_income: {
                   "$ref" => SCHEMA_COMPONENTS[:currency],
+                  # NB when using a $ref, the 'description' and 'deprecated' don't get displayed in the swagger web page - they are only visible to clients in the raw swagger.yaml
```

Comment:
I think I may have worked out that using oneOf: [ { "$ref" => SCHEMA_COMPONENTS[:currency] }] adds a constraint while allowing the description to still come through

Reply:
Nice tip - I've addressed this in https://github.com/ministryofjustice/cfe-civil/pull/265